ext/cairo/rb_cairo_font_face.c

```diff
@@ -108,6 +117,11 @@ rb_cairo_font_face_to_ruby_object (cairo_font_face_t *face)
 static VALUE
 cr_font_face_allocate (VALUE klass)
 {
+#if CAIRO_HAS_FT_FONT
+  FT_Error error = FT_Init_FreeType(&library);
```

Comment:
So I think there are two options: move the library to an ivar or move the allocate to happen globally for the library. The first will allow the library to live/die with the Ruby instance's lifecycle but is less memory efficient. The second basically means removing the free handler as the library would live until the process ends.